File: app-kotlin/src/main/java/com/TritiumGaming/phasmophobiaevidencepicker/views/BorderTextView.java
package com.TritiumGaming.phasmophobiaevidencepicker.views;

import android.content.Context;
import android.util.AttributeSet;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.constraintlayout.widget.ConstraintLayout;

import com.TritiumGaming.phasmophobiaevidencepicker.R;

public class BorderTextView extends ConstraintLayout {

    public BorderTextView(@NonNull Context context) {
        super(context);

        init(context);
    }

    public BorderTextView(@NonNull Context context, @Nullable AttributeSet attrs) {
        super(context, attrs);

        init(context);
    }

    public BorderTextView(@NonNull Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);

        init(context);
    }

    public BorderTextView(@NonNull Context context, AttributeSet attrs, int defStyleAttr, int defStyleRes) {
        super(context, attrs, defStyleAttr, defStyleRes);

        init(context);
    }

    public void init(Context context) {
        inflate(context, R.layout.layout_labeled_textview, this);
    }

}

File: app-view/src/main/java/com/tritiumgaming/phasmophobiaevidencepicker/presentation/ui/views/global/BorderTextView.kt
package com.tritiumgaming.phasmophobiaevidencepicker.presentation.ui.views.global

import android.content.Context
import android.util.AttributeSet
import androidx.constraintlayout.widget.ConstraintLayout
import com.tritiumgaming.phasmophobiaevidencepicker.R

class BorderTextView : ConstraintLayout {
    constructor(context: Context) :
            super(context) { init() }

    constructor(context: Context, attrs: AttributeSet?) :
            super(context, attrs) { init() }

    constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) :
            super(context, attrs, defStyleAttr) { init() }

    constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int, defStyleRes: Int) :
            super(context, attrs, defStyleAttr, defStyleRes) { init() }

    fun init() {
        inflate(context, R.layout.layout_labeled_textview, this)
    }
}
